skimage/measure/_regionprops.py

```diff
@@ -883,7 +919,6 @@ def _install_properties_docs():
     for p in [member for member in dir(RegionProperties)
               if not member.startswith('_')]:
         getattr(RegionProperties, p).__doc__ = prop_doc[p]
-        setattr(RegionProperties, p, property(getattr(RegionProperties, p)))
```

Comment:
@scikit-image/core this is the main controbiution of this PR. There seem to be quite a few people interested in region props PRs at this time, so it might be nice to get this one through!